File: ProjectEuler10/ProjectEuler10.py
def isPrime(number):
	for i in range(2, int((number ** 0.5) + 1)):
		if number % i == 0:
			return False
	return True

def main():
	s = 0
	for i in range(2, 2000000):
		if isPrime(i):
			s += i
	return s

print(main())

File: ProjectEuler10/ProjectEuler10.java
public class ProjectEuler10{
	
	public boolean isPrime(long number){
		int squareRoot = (int)Math.pow(number,0.5) + 1;
		for (int i = 2; i < squareRoot; i ++){
			if (number % i == 0){
				return false;
			}
		}
		return true;
	}

	public static void main(String[] args){
		long sum = 0;
		ProjectEuler10 call = new ProjectEuler10();
		for (long i = 2; i < 2000000; i++){
			if (call.isPrime(i)){
				sum += i;
			}
		}
		System.out.println(sum);
	}
}